src/services/query.js

```diff
@@ -621,6 +621,22 @@ ngeo.Query.prototype.doGetFeatureRequests_ = function(
         return;
       }
 
+      // add dimensions values
+      var dimensions = items[0].source.dimensions;
+      if (dimensions) {
+        for (var key in dimensions) {
+          // get the value from the global dimensions
+          var value = this.dimensions[key];
+          if (value === undefined) {
+            // get the value from the layer default value
+            value = dimensions[key];
+          }
+          if (value !== undefined) {
+            url = goog.uri.utils.setParam(url, key, value);
```

Comment:
Ok, can you explain, why do not use goog, what should I use instead ? Note that this comes from doGetFeatureInfoRequests_ existing code.
I see ``goog`` everywhere in this file.